src/command_spec.rs

```diff
@@ -1,11 +1,13 @@
 use std::convert::From;
+use std::ffi::OsString;
 use std::fmt::{Display, Formatter, Result};
 use std::process::Command;
 
 #[derive(Debug, PartialEq)]
 pub struct CommandSpec {
     pub executable: String,
     pub arguments: Vec<String>,
+    envs: Vec<(String, String)>,
```

Comment:
(Sry, didn't know where else to put this, since I can only comment on changed lines)

`impl From<&CommandSpec> for tokio::process::Command` also needs to take `envs` into account. `tokio::process::Command` implements `From<std::Process::Command>`, so we let's just use that.